src/__tests__/to-have-text-content.js

```diff
@@ -1,20 +1,37 @@
 import {render} from './helpers/test-utils'
 
-test('.toHaveTextContent', () => {
-  const {queryByTestId} = render(`<span data-testid="count-value">2</span>`)
+describe('.toHaveTextContent', () => {
+  test('handles positive test cases', () => {
+    const {queryByTestId} = render(`<span data-testid="count-value">2</span>`)
 
-  expect(queryByTestId('count-value')).toHaveTextContent('2')
-  expect(queryByTestId('count-value')).toHaveTextContent(2)
-  expect(queryByTestId('count-value')).toHaveTextContent(/2/)
-  expect(queryByTestId('count-value')).not.toHaveTextContent('21')
-  expect(() =>
-    expect(queryByTestId('count-value2')).toHaveTextContent('2'),
-  ).toThrowError()
+    expect(queryByTestId('count-value')).toHaveTextContent('2')
+    expect(queryByTestId('count-value')).toHaveTextContent(2)
+    expect(queryByTestId('count-value')).toHaveTextContent(/2/)
+    expect(queryByTestId('count-value')).not.toHaveTextContent('21')
+  })
 
-  expect(() =>
-    expect(queryByTestId('count-value')).toHaveTextContent('3'),
-  ).toThrowError()
-  expect(() =>
-    expect(queryByTestId('count-value')).not.toHaveTextContent('2'),
-  ).toThrowError()
+  test('handles negative test cases', () => {
```

Comment:
Good that you started separating the test cases. It was becoming hard to read. 👏 👏 👏